tools/chpl-language-server/src/bisect_compat.py

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2024-2024 Hewlett Packard Enterprise Development LP
+# Copyright 2024-2025-2025 Hewlett Packard Enterprise Development LP
```

Comment:
I'm going to let you check the rest of the tools directory and skip beyond them in the interest of time, but this one is also off.